Makefile

```
@@ -0,0 +1,84 @@
+#   Licensed under the Apache License, Version 2.0 (the "License");            #
+#   you may not use this file except in compliance with the License.           #
+#   You may obtain a copy of the License at                                    #
+#                                                                              #
+#       http://www.apache.org/licenses/LICENSE-2.0                             #
+#                                                                              #
+#   Unless required by applicable law or agreed to in writing, software        #
+#   distributed under the License is distributed on an "AS IS" BASIS,          #
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.   #
+#   See the License for the specific language governing permissions and        #
+#   limitations under the License.                                             #
+#                                                                              #
+# ---------------------------------------------------------------------------- #
+
+all: compile
+
+# Variables and Settings
+version     ?=  0.0.1
+target      ?=  ae
+org         ?=  aurae-runtime
+authorname  ?=  The Aurae Authors
+authoremail ?=  info@aurae.io
+license     ?=  Apache2
+year        ?=  2023
+copyright   ?=  Copyright (c) $(year)
+
+compile: mod ## Compile for the local architecture ⚙
+	@echo "Compiling..."
+	go build -ldflags "\
+	-X 'github.com/$(org)/$(target).Version=$(version)' \
+	-X 'github.com/$(org)/$(target).AuthorName=$(authorname)' \
+	-X 'github.com/$(org)/$(target).AuthorEmail=$(authoremail)' \
+	-X 'github.com/$(org)/$(target).Copyright=$(copyright)' \
+	-X 'github.com/$(org)/$(target).License=$(license)' \
+	-X 'github.com/$(org)/$(target).Name=$(target)'" \
+	-o bin/$(target) main.go
+	chmod +x bin/*
+
+mod: ## Go mod things
+	go mod tidy
+	go mod vendor
+	go mod download
+
+install: ## Install the program to /usr/bin 🎉
+	@echo "Installing..."
+	sudo cp $(target) /usr/bin/$(target)
+
+test: clean compile install ## 🤓 Run go tests
+	@echo "Testing..."
+	go test -v ./...
+
+clean: ## Clean your artifacts 🧼
+	@echo "Cleaning..."
+	rm -rvf release/*
+
+.PHONY: release
+release: ## Make the binaries for a GitHub release 📦
+	mkdir -p release
+	GOOS="linux" GOARCH="amd64" go build -ldflags "-X 'github.com/$(org)/$(target).Version=$(version)'" -o release/$(target)-linux-amd64 cmd/*.go
+	GOOS="linux" GOARCH="arm" go build -ldflags "-X 'github.com/$(org)/$(target).Version=$(version)'" -o release/$(target)-linux-arm cmd/*.go
+	GOOS="linux" GOARCH="arm64" go build -ldflags "-X 'github.com/$(org)/$(target).Version=$(version)'" -o release/$(target)-linux-arm64 cmd/*.go
+	GOOS="linux" GOARCH="386" go build -ldflags "-X 'github.com/$(org)/$(target).Version=$(version)'" -o release/$(target)-linux-386 cmd/*.go
+	GOOS="darwin" GOARCH="amd64" go build -ldflags "-X 'github.com/$(org)/$(target).Version=$(version)'" -o release/$(target)-darwin-amd64 cmd/*.go
```

```suggestion
	GOOS="linux" GOARCH="amd64" go build -ldflags "-X 'main.Version=$(version)'" -o release/$(target)-linux-amd64 cmd/*.go
	GOOS="linux" GOARCH="arm" go build -ldflags "-X 'main.Version=$(version)'" -o release/$(target)-linux-arm cmd/*.go
	GOOS="linux" GOARCH="arm64" go build -ldflags "-X 'main.Version=$(version)'" -o release/$(target)-linux-arm64 cmd/*.go
	GOOS="linux" GOARCH="386" go build -ldflags "-X 'main.Version=$(version)'" -o release/$(target)-linux-386 cmd/*.go
	GOOS="darwin" GOARCH="amd64" go build -ldflags "-X 'main.Version=$(version)'" -o release/$(target)-darwin-amd64 cmd/*.go
```
Same as other comment.